### PR TITLE
Fixes to pdf filenames and death certificate

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15303

*Description of changes:*
- Fix missing underscore in some PPR pdf filenames
- Refactor showDeathCertificate function to work as intended. Show deceased badge on TDTD flows only

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrInformation/useTransferOwners.ts [hunk 1]
```diff
@@ -21,9 +21,10 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
   ])
 
   /** Conditionally show DeathCertificate based on Transfer Type **/
-  const showDeathCertificate = (): boolean =>
-    getMhrTransferType.value?.transferType === ApiTransferTypes.SURVIVING_JOINT_TENANT ||
-    getMhrTransferType.value?.transferType === ApiTransferTypes.TO_EXECUTOR_PROBATE_WILL
+  const showDeathCertificate = (): boolean => {
+    return getMhrTransferType.value?.transferType === ApiTransferTypes.SURVIVING_JOINT_TENANT ||
+           getMhrTransferType.value?.transferType === ApiTransferTypes.TO_EXECUTOR_UNDER_25K_WILL
+  }
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/mhrInformation/useTransferOwners.ts:27]: thank you, i prefer this format as well :)

**Review by cameron-eyds — APPROVED** [review]
Looks good 👍 

I imagine this is required for merge asap for our today/tomorrow release?

**Comment by owillborn** [comment]
@cameron-eyds yes, planning to release PPR update so merge asap if we look good. @doug-lovett maybe quick scan when you have a minute?

**Review by doug-lovett — APPROVED** [review]
LGTM.

**Pull request merged**